### ISSUE TITLE
Dont use Crossed out Ban Pokemon in gym

### ISSUE THREAD

**Issue by judgejuctice** [pokesniperandroid/PokeSniper-Android#240]
Well we all know now that if you Sniped pokemon you get a red slash...Well I also found out if you use that Said Pokemon in a Gym Battle It will act up.  As in when i battled a Pokemon, the Pokemon that I attacked got healed in the process acting like someone is in the area healing the Pokemon. As in like the pokemon games if you don't have a Badge for that level 25 pokemon it will act weird.

**Comment by Ardhya94** [comment]
Yeah you're right .. currently we can't use slashed pokemon to fight again raid boss and also in game battle. 
If you still using this app, just follow softban cool down periode (about around >= 2 hours per catch... see jurnal for last catch / spin pokestop) and you'll be safe 😊
